tests/core/pyspec/eth2spec/test/capella/sanity/test_blocks.py

```diff
@@ -133,3 +139,69 @@ def test_exit_and_bls_change(spec, state):
     assert not spec.is_fully_withdrawable_validator(validator, balance, current_epoch)
     assert validator.withdrawable_epoch < spec.FAR_FUTURE_EPOCH
     assert spec.is_fully_withdrawable_validator(validator, balance, validator.withdrawable_epoch)
+
+
```

Comment:
We need to make sure we have a test that is at an epoch transition that induces balance changes for a to-be withdrawn validator

Reply:
From looking at the existing tests, several of them would already do this. For example `test_full_withdrawal_in_epoch_transition` sets the validator as withdrawable first, and then advances slots for an entire epoch which should affect balance of this validator.